voicevox_engine/app/routers/speaker.py

```diff
@@ -94,7 +140,7 @@ def _speaker_info(
 
             # speaker portrait
             portrait_path = speaker_path / "portrait.png"
-            portrait = b64encode_str(portrait_path.read_bytes())
+            portrait = manager.resource_str(portrait_path, self_url, resource_url)
```

Comment:
後ろ２つ長い＆何度も出てくるので、部分適用する局所関数定義すると便利かもです。
こんな感じ？
```suggestion
def _resource_str(path: Path):
  return manager.resource_str(path, self_url, resource_url)

portrait = _resource_str(portrait_path)
```

Reply:
局所関数を使用するようにしました。